### DISCUSSION TITLE
喝绿豆汤啦

### DISCUSSION THREAD

**Discussion by kallen1314** [share121/inter-knot#1382] [其他]
一边抱怨着天气热的要死一边守在炉灶旁边，等我弄好的时候，它开始刮凉风了，要下雨了，快得好像我在弄那什么求雨魔法
![IMG_20240811_152455](https://github.com/user-attachments/assets/e5702867-6b7a-42db-bdf3-e42db298d7e8)

**Comment by wwwwsn147** [comment]
分我一碗

**Comment by gaojiaobei** [comment]
这是很严重的问题，请分我两口

**Comment by WORLD-executeME** [comment]
我也要吃

**Comment by ZhuYuan-148** [comment]
好解暑

**Comment by 13579yangh** [comment]
114514区，蔡徐坤先生，对吗，您所在地区会出现空洞，请及时搬离

**Reply by qwq1949** [reply]
盒武器！

**Comment by kallen1314** [comment]
绿豆汤倒进模具做冰块了，问题来了，搞不下来……![1BAC1B8A79D87AF74AD76196512627CB](https://github.com/user-attachments/assets/2a27e5e3-5900-49c1-a4d9-6d1b6469467f)

**Reply by yaotian-yyds** [reply]
泡水里。

**Comment by SteveXXXXiao** [comment]
放一会就好了
